src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,42 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private final int maxSizeOfArray = 10;
```

Comment:
```suggestion
    private static final int MAX_SIZE = 10;
```
краще цю змінну зробити константою так як вона в процесі не повинна змінюватись за поточним завданням